app/controllers/schools/advice/base_long_term_controller.rb

```diff
@@ -0,0 +1,87 @@
+module Schools
+  module Advice
+    class BaseLongTermController < AdviceBaseController
+      def insights
+        @analysis_dates = analysis_dates
+        @annual_usage = usage_service.annual_usage
+        @vs_benchmark = usage_service.annual_usage_vs_benchmark(compare: :benchmark_school)
+        @vs_exemplar = usage_service.annual_usage_vs_benchmark(compare: :exemplar_school)
+        @annual_usage_change_since_last_year = usage_service.annual_usage_change_since_last_year
+        @benchmarked_usage = benchmarked_usage(@annual_usage.kwh)
+      end
+
+      def analysis
+        @analysis_dates = analysis_dates
+
+        @annual_usage = usage_service.annual_usage
+        @vs_benchmark = usage_service.annual_usage_vs_benchmark(compare: :benchmark_school)
+        @vs_exemplar = usage_service.annual_usage_vs_benchmark(compare: :exemplar_school)
+
+        @estimated_savings_vs_exemplar = usage_service.estimated_savings(versus: :exemplar_school)
+        @estimated_savings_vs_benchmark = usage_service.estimated_savings(versus: :benchmark_school)
+
+        @multiple_meters = multiple_meters?
+        if @multiple_meters
+          @annual_usage_meter_breakdown = usage_service.annual_usage_meter_breakdown
+          @meters_for_breakdown = sorted_meters_for_breakdown(@annual_usage_meter_breakdown)
+        end
+      end
+
+      private
+
+      def multiple_meters?
+        @school.meters.active.where(meter_type: fuel_type).count > 1
+      end
```

Comment:
not something to change in this pr but maybe this could be a method on the meters or school model?